src/Focus.js

```diff
@@ -5,15 +5,25 @@ import './Focus.css'
 
 function Focus(props) {
   const [selectedFish, setSelectedFish] = useState({})
-  const [savedBool, setSavedBool] = useState(false);
 
   const {name} = useParams();
 
   useEffect(() => {
     getSingleFish(name)
     .then(data => setSelectedFish(data))
```

Comment:
Now passing the setError(error) state through your Focus.js component. Looks good!